src/cli/completion.rs

```diff
@@ -272,7 +311,18 @@ _arguments "${_arguments_options[@]}" \
     --help(-h)                # Print help (see more with '--help')
   ]"#;
         let result = replace_nushell_completion(script);
-        insta::assert_snapshot!(result);
+        let replacement = format!("{} run", pixi_consts::consts::PIXI_BIN_NAME.as_str());
+        let nu_complete_run = format!(
+            "nu-complete {} run",
+            pixi_consts::consts::PIXI_BIN_NAME.as_str()
+        );
+        println!("{}", result);
+        insta::with_settings!({filters => vec![
+            (replacement.as_str(), "[PIXI RUN]"),
+            (nu_complete_run.as_str(), "[nu_complete_run PIXI COMMAND]"),
+        ]}, {
+            insta::assert_snapshot!(result);
+        });
     }
```

Comment:
I don't really understand why you need to adapt this test.

Reply:
This is a weird one because of how cargo configs work where it recurses parent directories. It picks up on cargo a config from a higher level folder so when I run the test with magic the name when running the test changes. I solved this actually a different way by moving magic out of the parent path but the issue is still there if someone else does it or they explicitly override the name of exe when building with cargo. This fixes this with filter substitutions in insta.